packages/shared/tests/schema/expoJudgingVote/post.ts

```diff
@@ -0,0 +1,54 @@
+import { Schema } from '../../../src';
+
+const validExpoJudgingVote = {
+  currentProjectChosen: true,
+  expoJudgingSessionId: '1',
+};
+
+describe('expo judging vote post schema', () => {
+  it('validates a matching object', () => {
+    expect(Schema.project.post.safeParse(validExpoJudgingVote).success).toBe(true);
+  });
+
+  it('does not validate an object with an invalid currentProjectChosen', () => {
+    expect(
+      Schema.project.post.safeParse({
+        ...validExpoJudgingVote,
+        currentProjectChosen: 'invalid',
+      }).success,
+    ).toBe(false);
+  });
+
+  it('does not validate an object with an invalid expoJudgingSessionId', () => {
+    expect(
+      Schema.project.post.safeParse({
+        ...validExpoJudgingVote,
+        expoJudgingSessionId: 'invalid',
```

Comment:
This might need an `as any`
```suggestion
        expoJudgingSessionId: 1,
```